src/services/statistic/index.js

```diff
@@ -100,10 +95,14 @@ const promises = [
 const fetchStatistics = () => {
 	const statistics = {};
 
-	return Promise.all(promises.map((p) => p.promise.exec().then((res) => {
-		statistics[p.name] = res;
-		return res;
-	}))).then((_) => statistics);
+	return Promise.all(
+		promises.map((p) =>
+			p.promise.exec().then((res) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected no linebreak before this expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5400801)